bam_consensus.c

```diff
@@ -1557,12 +2407,18 @@ int main_consensus(int argc, char **argv) {
         {"min-MQ",             required_argument, NULL, 13},
         {"min-BQ",             required_argument, NULL, 16},
         {"P-het",              required_argument, NULL, 15},
+        {"P-indel",            required_argument, NULL, 17},
+        {"het-scale",          required_argument, NULL, 19},
         {"mode",               required_argument, NULL, 'm'},
         {"homopoly-fix",       no_argument,       NULL, 'p'},
+        {"homopoly-score",     required_argument, NULL, 'p'+100},
+        {"homopoly-redux",     required_argument, NULL, 'p'+200},
```

Comment:
What does this do?  It doesn't appear to be documented anywhere.

Reply:
Deliberately so.  They're experimental and don't appear in the documentation yet.